<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1206</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>850</width>
     <height>450</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="full_container">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">
QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 1px solid #C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 4px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: white;
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: white; /* same as pane color */
}

QTabBar::tab:!selected {
    margin-top: 2px; /* make non-selected tabs look smaller */
}

/* make use of negative margins for overlapping tabs */
QTabBar::tab:selected {
    /* expand/overlap to the left and right by 4px */
    margin-left: -4px;
    margin-right: -4px;
}

QTabBar::tab:first:selected {
    margin-left: 0; /* the first selected tab has nothing to overlap with on the left */
}

QTabBar::tab:last:selected {
    margin-right: 0; /* the last selected tab has nothing to overlap with on the right */
}

QTabBar::tab:only-one {
    margin: 0; /* if there is only one tab, we don't want overlapping margins */
}</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pokedexTab">
         <attribute name="title">
          <string>POKEDEX</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="layout_pokemonTable">
            <item>
             <widget class="QTableWidget" name="pokemonTable">
              <property name="maximumSize">
               <size>
                <width>1000000</width>
                <height>1000000</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QHeaderView::section {
    background-color: #bebebe;
    color: black;
    border: 0px;
    font-weight: bold;
}

QScrollBar::vertical{
	width:15px;
    background-color: #bebebe;
	border-radius:4px;
}


QListView {
    border:1px black solid ; 
}

</string>
              </property>
              <property name="alternatingRowColors">
               <bool>false</bool>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="layout_filters">
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="topMargin">
               <number>10</number>
              </property>
              <property name="rightMargin">
               <number>10</number>
              </property>
              <property name="bottomMargin">
               <number>10</number>
              </property>
              <item>
               <widget class="QLabel" name="scegli_tipi">
                <property name="font">
                 <font>
                  <pointsize>9</pointsize>
                  <italic>false</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Scegli i Tipi:</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="filtersLayout"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="compareTab">
         <attribute name="title">
          <string>CONFRONTA</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="layout_compareTab">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="pokemon1_layout">
              <item>
               <widget class="QComboBox" name="pokemon1_combobox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="layout_pokemon1_img">
                <property name="leftMargin">
                 <number>10</number>
                </property>
                <property name="topMargin">
                 <number>10</number>
                </property>
                <property name="rightMargin">
                 <number>10</number>
                </property>
                <property name="bottomMargin">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QLabel" name="pokemon1_img">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>9</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Nessun Pokemon Selezionato</string>
                  </property>
                  <property name="textFormat">
                   <enum>Qt::AutoText</enum>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="pokemon1_types"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="layout_chart">
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="topMargin">
               <number>10</number>
              </property>
              <property name="rightMargin">
               <number>10</number>
              </property>
              <property name="bottomMargin">
               <number>10</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="chart_layout">
                <property name="sizeConstraint">
                 <enum>QLayout::SetDefaultConstraint</enum>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="pokemon2_layout">
              <item>
               <widget class="QComboBox" name="pokemon2_combobox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="layout_pokemon2_img">
                <property name="leftMargin">
                 <number>10</number>
                </property>
                <property name="topMargin">
                 <number>10</number>
                </property>
                <property name="rightMargin">
                 <number>10</number>
                </property>
                <property name="bottomMargin">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QLabel" name="pokemon2_img">
                  <property name="font">
                   <font>
                    <pointsize>9</pointsize>
                    <italic>false</italic>
                    <bold>true</bold>
                    <underline>false</underline>
                    <strikeout>false</strikeout>
                    <kerning>true</kerning>
                   </font>
                  </property>
                  <property name="text">
                   <string>Nessun Pokemon Selezionato</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="pokemon2_types"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1206</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
